tox.ini

```diff
@@ -41,6 +41,8 @@ whitelist_externals = make
 
 [testenv:py26-django14]
 basepython = python2.6
+install_command = pip install --no-binary=Django {opts} {packages}
```

Comment:
Explained in the commit message https://github.com/moreati/django-two-factor-auth/commit/b7d63814876f9d88aa201c59552ab6098599419c

> Recent versions of pip and virtualenv default to installing packages as
wheels. This fails for Django 1.4, the templates are installed to the
root of the virtualenv, rather than `.../site-packages/django/....` When
the user_session unit tests are run no templates can be found.